extension/chrome/elements/passphrase.ts

```diff
@@ -35,7 +35,7 @@ View.run(class PassphraseView extends View {
     Ui.event.protect();
     await initPassphraseToggle(['passphrase']);
     const allPrivateKeys = await KeyStore.get(this.acctEmail);
-    this.myPrivateKeys = allPrivateKeys.filter(ki => this.longids.includes(ki.longid) || (ki.primary && this.longids.includes('primary')));
+    this.myPrivateKeys = allPrivateKeys.filter(ki => this.longids.includes(ki.longid));
```

Comment:
I just noticed, this `myPrivateKeys` has unfortunate name. It should say `keysWeNeedPassPhraseFor` or something like that. Please help me rename it.